src/VocabGeneration.test.js

```diff
@@ -101,15 +128,15 @@ const ConfigSolid = {
 };
 
 describe("Suite for generating common vocabularies (marked as [skip] to prevent non-manual execution", () => {
-  // it("Generate ALL vocabs recursively", async () => {
-  it.skip("Generate ALL vocabs recursively", async () => {
+  // it("Generate ALL vocabs", async () => {
+  it.skip("Generate ALL vocabs", async () => {
     jest.setTimeout(6000000);
     await generateVocabArtifact(ConfigAll);
   });
 
-  // it("Generate ALL vocabs", async () => {
-  it.skip("Generate ALL vocabs", async () => {
-    jest.setTimeout(120000);
+  // it("Generate ALL specific vocabs", async () => {
+  it.skip("Generate ALL specific vocabs", async () => {
+    jest.setTimeout(1200000);
```

Comment:
this is 20 minutes, which seems excessive unless something actually has gone wrong (how long do we expect?)

Reply:
Yeah :) - it can take over 10 minutes (on my local machine), as we're generating so many variations for both Java and JavaScript artifacts now. It probably doesn't need to be 20 mins alright - I just kept doubling the value as the number of artifacts grew and grew, but since this is a 'test' (as are all the 'tests' in this test file) that I only ever run manually anyway, I'm kinda grand keeping the value 'plenty high', as it's a real pain to see it chugging away for ages and then just 'failing' 'cos the timeout expired.
Good to point out though :) !